Scripts/CommonServerPython/CommonServerPython.yml

```diff
@@ -16,4 +16,4 @@ timeout: 0s
 alt_dockerimages: # used for unit testing
```

Comment:
Why is a change to CommonServerPython.yml file here? I thought you reverted the CommonServerPython related changes.

Reply:
reverting